### PR TITLE
Update ja_JP.lang

### PR DESCRIPTION
**Pull request by neconeco2**
- Add new item name.
- Fixed a bit.

### CODE REVIEW

**Comment by TehNut** [comment]
There are some new blocks that still need to be localized for US. I'll pull this after we get those done so you can do them

**Comment by neconeco2** [comment]
Yes , I understood. thank you.

**Comment by WayofTime** [comment]
<3

**Comment by TehNut** [comment]
Please update with the latest changes and I'll pull it.

File: src/main/resources/assets/bloodmagic/lang/ja_JP.lang [hunk 1]
```diff
@@ -647,7 +669,7 @@ chat.BloodMagic.ritual.notValid=あなたはルーンの配置が正しくない
 
 chat.BloodMagic.livingArmour.upgrade.poisonRemove=あなたは良い効果を感じた！
 chat.BloodMagic.livingArmour.upgrade.fireRemove=&6清涼感があなたを包み込み、炎上が収まった。
-chat.BloodMagic.livingArmour.upgrade.grimReaper=&6闇の力があなたを死のふちに引き込む！
+chat.BloodMagic.livingArmour.upgrade.grimReaper=&6闇の力があなたを死の淵から引き寄せる！
```
Comment (neconeco2) [inline src/main/resources/assets/bloodmagic/lang/ja_JP.lang:672]: mistake of translation. ><

**Comment by neconeco2** [comment]
Well... en_US.lang line 280
tile.BloodMagic.bricks2.tile_raw.name=Small Raw Stone Tiles
↑ This is "Raw Stone Tiles" ?

**Comment by TehNut** [comment]
Yes that seems to be wrong. Shouldn't be "Small".

Is this ready to pull?

**Comment by neconeco2** [comment]
ok

**Pull request merged**